### DISCUSSION TITLE
"Your account has been marked as spam and therefore cannot authorize a third party application" - Please help. Request #2803361

### DISCUSSION THREAD

**Discussion by himanshuSri24** [community/community#127280] [Other Feature Feedback, Questions, & Ideas] [answered]
My friend's account has been marked as spam and he's unable to access anything on it. We had been working on projects together and can no longer do it.

He raised a ticket as well and it's been more than 2 weeks, yet no reply.

We're not sure what to do, and really need to get the account back.

Please help?

We can't raise another ticket since it tells a duplicate request exists

There's no extra email ID added as well, just the primary email ID

Affected github username: HarshitTib

**Comment by ebndev** [comment, marked as answer]
Hi @himanshuSri24,

This community is managed by GitHub employees, but it is not a support forum. Your friend has done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
